### DISCUSSION TITLE
Templates and Values in different repos via ArgoCD

### DISCUSSION THREAD

**Discussion by hutger** [argoproj/argo-cd#10024] [Q&A]
I'm looking for insights for the following situation...

 - I have one ArgoCD application pointing to a Git repo (A), where there's a `values.yaml`;
 - I would like to use the Helm templates stored in a different repo (B);

Any suggestions/alternatives on how to make this work?

Thanks in advance.

**Comment by crenshaw-dev** [comment]
First-class support for this use case will land in 2.5 (~August 2022).

Until then, quite a few workarounds are documented here: https://github.com/argoproj/argo-cd/issues/2789

**Reply by hutger** [reply]
Hi @crenshaw-dev do we have ETA when 2.5 will be in GA?
